LICENSE

```diff
@@ -0,0 +1,19 @@
+MIT License
```

Comment:
This is fine for now.  Eventually I need to decide if this is CC BY (like the handbook) or MIT + Apache (like our other repos) or fine being just MIT (which is different from our other repos).